packages/editor-ui/package.json

```diff
@@ -44,6 +44,7 @@
     "@vueuse/components": "^10.11.0",
     "@vueuse/core": "^10.11.0",
     "axios": "catalog:",
+    "bowser": "2.11.0",
```

Comment:
Could we implement this without adding additional dependency just to detect Safari?

Reply:
1. This is already a transitive dependency (check the lockfile changes)
2. This adds about 5KB to the bundle before any tree-shaking